docs/sources/whatsnew/whats-new-in-v10-1.md

```diff
@@ -0,0 +1,498 @@
+
+{{< figure src="/media/docs/alerting/alert-routing-preview.png" max-width="750px" caption="Alert instance routing preview" >}}
+
+#### Alert rule types
+
+Enables you to switch to a data source-managed alert rule if your data source is configured to support alert rule creation (Ruler API enabled). By default, the alert rule type is Grafana-managed.
+
+{{< figure src="/media/docs/alerting/alert-rule-types.png" max-width="750px" caption="Alert rule types" >}}
+
+#### UI improvements
+
+- **Alert evaluation behavior**: New UI components for creating a folder and adding an evaluation group along with improved text and validation.
+- **Alert Rule list page**: The process of creating recording rules (**More** drop-down) is now separate from Grafana-managed and data source-managed alert rules (**+New alert rule**)
+  .
+- **Annotations display**: Adding a summary, description, and runbook URL as annotations are now optional. The dashboard and panel names are now also linked directly, making it easier to access.
+- **View YAML button**: Displays alert rule configuration in YAML format on the Grafana-managed alert rules form, as well as the Grafana-managed provisioned and non-provisioned Alert Rule detail view.
+- **Queries and expressions**: Several improvements have been made to the display of queries and expressions, including making **Add expression** a drop-down and moving **Conditions** to the header.
+- **Min interval option**: Improves control over query costs and performance by allowing you to adjust the minimum resolution of the data used in your alerting queries.
+- **In-app guidance for alert rule creation**: Learn about how to create your alert rules interactively with in-app guidance for additional context and links out to our Alerting documentation.
+- **Support for toggling common labels**: Toggle between showing or hiding labels for each individual alert instance.
+
+### Contact points
+
+We’ve made the following changes to contact points.
+
+#### Additional contact points for external Alertmanager
+
+Adds support for the following contact points when using an external Alertmanager:
+
+- WeChat
+- Amazon SNS
+- Telegram
+- Cisco Webex Teams
+
+#### Contact point provisioning file export
+
+Facilitates file provisioning and maintenance for contact points. This feature implements the provisioning API export endpoints for exporting contact points as well as adding export buttons to the contact point list in the UI.
+
+### Notification policies
+
+We’ve made the following changes to notification policies.
+
+#### Notification policy provisioning file export
+
+Facilitates file provisioning and maintenance for notification policies. This feature implements the provisioning API export endpoints for exporting notification policies as well as adding an export button to the root notification policy in the UI.
```

Comment:
```suggestion
This update facilitates file provisioning and maintenance for notification policies. The feature implements the provisioning API export endpoints for exporting notification policies as well as adding an export button to the root notification policy in the UI.
```

Reply:
fine for me. again though-this adds extra words that we don't necessarily need as verb is always more direct...but i'm happy to be consistent and go with the style we decide on:)